### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by frankrousseau**
**Problem**

* The concept side bar can't be displayed in the asset page
* Assets are not properly loaded when getting into the assets page from the production list

**Solution**

* Show the side bar when a concept is selected.
* Set a default value to the current section in the top bar

### CODE REVIEW

**Review by NicoPennec — CHANGES_REQUESTED** [review]
The following commit "[[concepts] Display concept side bar in the asset page](https://github.com/cgwire/kitsu/pull/1504/commits/60248732011a81aea76b6cf8bb068f44fb6d9831)" is not consistent.
Changes related to the `@click` action from ConceptCard.vue are missing?

**Comment by frankrousseau** [comment]
@NicoPennec I don't understand what you mean. Can you make it clearer?

File: src/components/widgets/ConceptCard.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="concept-item">
+  <div class="concept-item" @click="$emit('click')">
```
Comment (NicoPennec) [inline src/components/widgets/ConceptCard.vue:2]: This `$emit('click')` seems useless, as it was never used in the application.

Comment (frankrousseau) [reply src/components/widgets/ConceptCard.vue:2]: Ok thank you, a commit was missing.

**Comment by EvanBldy** [comment]
Thanks !

**Pull request merged**